projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -755,7 +748,7 @@ function JetpackContactFormEdit( { name, attributes, setAttributes, clientId, cl
 						/>
 					</PanelBody>
 
-					{ ! isSimpleSite() && canUserInstallPlugins && (
+					{ showFormIntegrations && (
```

Comment:
The main change. We no longer check ifSimpleSite or check permissions. We just the feature flag. Any given environment (WordPress.com, CIAB, VIP) is now responsible for turning this off if needed.